documentation/projects/proposals/additional_search_views/20230719-implementation_plan_additional_search_views.md

```diff
@@ -111,35 +118,156 @@ for fuzzy matches.
 **Finally**, we apply the ranking and sorting parameters, and "highlight" the
 fields that were matched in the results.
 
-The new search controller should allow for using different filters for the first
-step and to not use the full-text search. We should also create a new serializer
-for collection search requests. It should include the common parameters for
-`list` requests, such as `page` and `page_size`, and the parameters for the
-exact matches: `tag`, `creator` and `source`.
+The search controller needs to be updated to extract the first 2 steps into
+`build_search_query` method. A new `build_collection_query` should be added and
+used when the `collection` parameter is present. This method should create a
+filter query for the relevant field and value.
+
+The pagination and dead link cleanup should be the same for additional search
+views as for the default search views.
```

Comment:
```suggestion
The pagination and dead link cleanup should be the same for additional search
views as for the default search views.
```

Should this read "queries" instead of "views", since there will no longer be dedicated API endpoints for the collections?